pkg/cloudprovider/providers/vsphere/OWNERS

```diff
@@ -1,7 +1,7 @@
-maintainers:
-- dagnello
+approvers:
 - abithap
```

Comment:
Can we drop abithap and dagnello, from what I understand they are no longer working on this.
@abithap @dagnello